OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs

```diff
@@ -72,13 +72,18 @@ public IEnumerable<IRenderable> RenderAfterWorld(WorldRenderer wr)
 			if (targets == null || targets.Count == 0)
 				yield break;
 
+			var validTargets = new List<WPos>();
+			validTargets.Add(self.CenterPosition);
+
 			foreach (var target in targets)
 			{
 				if (target.Type == TargetType.Invalid)
 					continue;
 
-				yield return new TargetLineRenderable(new[] { self.CenterPosition, target.CenterPosition }, c);
+				validTargets.Add(target.CenterPosition);
 			}
+
+			yield return new TargetLineRenderable(validTargets, c);
```

Comment:
No I didn't know Enumerable.Empty<TargetLineRenderable>() existed, I also didn't know that c sharp allowed the array to inherit they type in this case.